tests/pub/data/100/images.json

```diff
@@ -32,6 +32,14 @@
     "state": "PUSHED", 
     "type": "hourly", 
     "virtualization": "hvm", 
-    "volume": "gp2"
+    "volume": "gp2",
```

Comment:
Similarly to the comments about the staging area testdata:

100% of pub tasks which exist today are not going to have these new fields in images.json.

So, can we please keep at least one test which covers the case where all of those fields are absent?

Reply:
I have removed newly added metadata from 100/images.json